apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsArrayControl.tsx

```diff
@@ -275,27 +290,22 @@ export function JsonFormsArrayControl({
           path={composePaths(path, `${selectedIndex}`)}
           label={label}
           setSelectedIndex={setSelectedIndex}
+          isRemoveItemDisabled={minItems !== undefined && data <= minItems}
+          handleRemoveItem={handleRemoveItem(path, selectedIndex)}
         />
       )}
 
-      {selectedIndex === undefined ? (
-        <Button
-          onClick={addItem(path, createDefaultValue(schema, rootSchema))}
-          w="100%"
-          variant="outline"
-          isDisabled={maxItems !== undefined && data >= maxItems}
-        >
-          Add item
-        </Button>
-      ) : (
-        <Button
-          onClick={handleRemoveItem(path, selectedIndex)}
-          variant="clear"
-          colorScheme="critical"
-          isDisabled={minItems !== undefined && data <= minItems}
-        >
-          Remove item
-        </Button>
+      {selectedIndex === undefined && (
+        <>
```

Comment:
nit: unnecessary fragment

Reply:
Updated in 7f2ee305b59a98499427b90037631637e6b43895.